api/app/routes/chat.py

```
@@ -41,7 +41,7 @@
 async def query(
     request: Request,
     settings: Settings = Depends(get_settings),
-    body_payload: dict = Body(...)
+    body_payload: dict = Body(...),
```

Ruff (0.12.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)